nexus/db-queries/src/db/queries/region_allocation.rs

```diff
@@ -502,18 +511,19 @@ mod test {
         let conn = pool.pool().get().await.unwrap();
 
         let volume_id = Uuid::new_v4();
-        let block_size = 512;
-        let blocks_per_extent = 4;
-        let extent_count = 8;
+        let params = RegionParameters {
+            block_size: 512,
+            blocks_per_extent: 4,
+            extent_count: 8,
+            read_only: false,
```

Comment:
Do we need a test where `read_only` is `true`?
This is really just a field in the database, right?  It does not change how
the region itself is created?

Reply:
> It does not change how the region itself is created?

Correct! This query will allocate a region where there's available space, regardless of whether it's read_only or not. The `read_only` parameter is inserted into the new `region` record during the data modifying part of the query.

> Do we need a test where read_only is true?

The tests in this file test the expected text of the region allocation query, and the value of `read_only` doesn't change that, so we don't need an additional test where `read_only` is true.